metacat-thrift/src/main/java/com/netflix/metacat/thrift/HiveConvertersImpl.java

```diff
@@ -282,6 +283,9 @@ private StorageDescriptor fromStorageDto(@Nullable final StorageDto storageDto,
         // Setting an empty skewed info.
         result.setSkewedInfo(new SkewedInfo(Collections.emptyList(), Collections.emptyList(), Collections.emptyMap()));
         result.setParameters(sdParams);
+        result.setSkewedInfo(new SkewedInfo(new ArrayList<String>(),
```

Comment:
We can remove this. We are already doing this above.

Reply:
Will do.